src/dealDashboard/discussionThread/singleComment/singleComment.ts

```diff
@@ -26,42 +16,53 @@ export class SingleComment {
     down: false,
   };
 
+  @bindable public author: string;
```

Comment:
I assume this is not really your change.  Some kind of merging problem?

Reply:
There is nothing changed in here just ordered and bindables made public to represent what they actually are.

Reply:
cool, and thanks, but our practice is that changes unrelated to what is being addressed by a PR should be done in a separate PR.  Can you please create a ticket for this and put it in the Triage column (the default "pipeline") of the kanban, and back the change out of this PR?